### ISSUE TITLE
Test the writeto method of HDUList fails

### ISSUE THREAD

**Issue by olebole** [spacetelescope/stsci.tools#50]
After quickfixing #48 (by removing the keyword) and #47, I get still the following errors:

```
======================================================================
FAIL: Test the writeto method of HDUList in both the fits and stpyfits
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/stsci.tools-3.4.11/.pybuild/pythonX.Y_2.7/build/stsci/tools/tests/testStpyfits.py", line 435, in testHDUListWritetoMethod
    assert_equal(hdul1[1].header['NAXIS'], 2)
AssertionError: 0 != 2

======================================================================
FAIL: Test the update convience function in both the fits and stpyfits
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/stsci.tools-3.4.11/.pybuild/pythonX.Y_2.7/build/stsci/tools/tests/testStpyfits.py", line 305, in testupdateConvienceFunction
    assert_equal(hdul7[1].header['NAXIS'], 2)
AssertionError: 0 != 2

```

**Comment by pllim** [comment]
TBH I don't think these tests are run very often, if at all. I'll make an issue to enable Travis CI here to ensure this problem won't repeat itself in the future. However, I don't have any timeline on when that can happen. Are failing tests a show-stopper for your work? Can you maybe just `xfail` them for now until we have time to sort this out on our side? I apologize for any inconvenience caused.

**Comment by olebole** [comment]
I need the package mainly for pyraf; however it will be installed as a general package in Debian. I created the issues mainly to see how important theproblems are. So,as long as you don't veto like "don't use with astropy-2", I will just xfail these tests.

**Comment by pllim** [comment]
For now, Python 2 is still theoretically supported. However, we have plans to move exclusively to Python 3 in the near future. Perhaps @stsci-hack could comment more on this when he is back.

**Comment by olebole** [comment]
I meant __astropy__ 2.0 :-) For Py2->Py3, I will just follow you; We (Debian) also want to get rid of Python 2. So, if you think Pyraf is stable enough with Python 3 I will not create a Python 2 package for it (and also not Python 3 stsci.tools package).

**Comment by cdsontag** [comment]
Hi all!  PyRAF should be fine with Python3 (has been tested on it for 5 years and many users do use it), but recall that the codebase is 2-to-3-able-Python2 (for now).  So just make sure the install process (or maybe your packaging process) 2to3's it first.

See the FAQ for some (slightly outdated) info:
- http://www.stsci.edu/institute/software_hardware/pyraf/pyraf_faq#1.8

**Comment by pllim** [comment]
Update: I am looking into the failures you reported.

**Comment by pllim** [comment]
Should have been fixed by #61. If you still see this problem, feel free to re-open.
